src/components/create-user.html

```diff
@@ -44,31 +42,90 @@
       }
     </style>
     <h3>Create User Component</h3>
-    <form>
-      <fieldset>
-        <div>
-          <label for="firstName">First name:</label>
-          <input type="text" name="firstName" id="firstName">
-        </div>
-        <div>
-          <label for="lastName">Last name:</label>
-          <input type="text" name="lastName" id="lastName">
-        </div>
-        <div>
-          <label for="phone">Phone:</label>
-          <input type="text" name="phone" id="phone">
-        </div>
-        <div>
-          <label for="email">Email:</label>
-          <input type="text" name="email" id="email">
-        </div>
-        <button type="submit">Save</button>
-      </fieldset>
-    </form>
+    <template is="dom-if" if="[[addUserSelected]]">
```

Comment:
This seems like it should be opposite. Like when you clicked the button that means you've selected addUser.

Reply:
yeah I can change that, it makes more sense that way for sure